plugins/woocommerce-admin/client/marketplace/utils/functions.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Internal dependencies
+ */
+import { Product } from '../components/product-list-content/product-list-content';
+
+interface ProductCategory {
+	id: number;
+	title: string;
+	items: Product[];
+}
+
+// Fecth data for the discover page from the WooCommerce.com API
+const fetchDiscoverPageData = async (): Promise< ProductCategory > => {
```

Comment:
I'm confused, we seem to be expecting a single returned item here – doesn't the endpoint return an array of groups of products, and so not a single thing? If we rename `ProductCategory` to `ProductGroup`, wouldn't this be

```suggestion
const fetchDiscoverPageData = async (): Promise< Array< ProductGroup >> => {
```

Reply:
Yes, it should be the `Array`. 👍